src/routes/donate/+page.svelte

```diff
@@ -0,0 +1,328 @@
+<script lang="ts">
+	import { fly, slide } from 'svelte/transition';
+	import { quintOut } from 'svelte/easing';
+	import Meta from '$lib/components/Meta.svelte';
+	import Footer from '$layout/Footer/FooterHost.svelte';
+	import Button from '$lib/components/Button.svelte';
+	import Snackbar from '$lib/components/Snackbar.svelte';
+	import QRCode from './QRCode.svelte';
+
+	import { queries } from '$data/api';
+	import { createQuery } from '@tanstack/svelte-query';
+	import Query from '$lib/components/Query.svelte';
+	import Dialogue from '$lib/components/Dialogue.svelte';
+
+	let qrCodeDialogue = false;
+	let qrCodeValue = '';
+	let qrCodeDialogueName = '';
+	const teamQuery = createQuery(['team'], queries.team);
+	const donateQuery = createQuery(['donate'], queries.donate);
+	let snackbarOpen = false;
+	let expandCrypto = false;
+
+	async function copyToClipboard(walletAddress: string) {
+		snackbarOpen = true;
+		qrCodeDialogue = false;
+		try {
+			await navigator.clipboard.writeText(walletAddress);
+		} catch (error) {
+			console.error('Failed to copy crypto wallet:', error);
+		}
+	}
+</script>
+
+<Meta title="Donate" />
+
+<main class="wrapper" in:fly={{ y: 10, easing: quintOut, duration: 750 }}>
+	<h2>🎉 Support <span style="color: var(--accent-color);">ReVanced</span></h2>
+	<p>
+		ReVanced offers a variety of patches, including ad-blocking, custom themes, and innovative
+		features. All of which is completely open source and free of charge. Donating will allow
+		ReVanced maintain our servers and develop new features.
+	</p>
+	<Query query={donateQuery} let:data>
+		{#if data.platforms}
+			<div class="buttons-container">
+				{#each data.platforms as platform, i}
+					{#if i === 0}
+						<Button type="filled" href={platform.url}>{platform.name}</Button>
+					{:else}
+						<Button type="tonal" href={platform.url}>{platform.name}</Button>
+					{/if}
+				{/each}
+			</div>
+		{/if}
+		{#if data.wallets}
+			<div class="crypto-card">
+				<div
+					class="crypto-title"
+					class:closed={!expandCrypto}
+					on:click={() => (expandCrypto = !expandCrypto)}
+					on:keypress={() => (expandCrypto = !expandCrypto)}
+				>
+					<h4>Crypto</h4>
+					<img
+						id="arrow"
+						style:transform={expandCrypto ? 'rotate(0deg)' : 'rotate(-180deg)'}
+						src="/icons/expand_less.svg"
+						alt="dropdown"
+					/>
+				</div>
+				{#if expandCrypto}
+					<hr />
+					<div class="wallets" transition:slide|local={{ easing: quintOut, duration: 500 }}>
+						{#each data.wallets as wallet}
+							<button
+								on:click={() => {
+									qrCodeValue = wallet.address;
+									qrCodeDialogueName = wallet.currency_code;
+									qrCodeDialogue = !qrCodeDialogue;
+								}}
+							>
+								<div class="name">
+									<img src="/donate/{wallet.currency_code}.svg" alt={`${wallet.network} icon.'`} />
```

Comment:
How do you handle currencies that you do not have an SVG for? You should default to a generic image.